content/enterprise/releases/2022/v202203-1.mdx

```diff
@@ -0,0 +1,39 @@
+---
+page_title: Releases - Terraform Enterprise
+---
+
+# TFE Release v202203-1 (607)
+
+## UPCOMING DEPRECATION NOTICES:
+
+The Terraform Enterprise April 2022 release will:
+
+1. Remove the [demo operational mode](https://www.terraform.io/enterprise/before-installing#operational-mode-decision), which is also known as the proof of concept (PoC) operational mode. The mounted disk operational mode will replace the demo operational mode for both non-production and production Terraform Enterprise environments. To check which mode your installation is using, run `replicatedctl app-config export --template '{{ .installation_type.Value }}'`. The value `poc` indicates that your installation is using the demo operational mode. The April 2022 release notes will contain more information about how to migrate.
+1. Update the names of containers, which may break container monitoring or custom tooling that identifies containers by name. The April 2022 release notes will explain these name changes in more detail and provide a complete list of old and new container names.
+1. Change the default value of [`restrict_worker_metadata_access`](https://www.terraform.io/enterprise/install/automated/automating-the-installer#restrict_worker_metadata_access) to 1 (true) instead of 0 (false). If you rely on the instance metadata endpoint (and make use of its instance profile), you must explicitly set the `restrict_worker_metadata_access` configuration flag in `replicated.conf` to 0.
+
+## APPLICATION LEVEL BREAKING CHANGES:
+
+1. The minimum required Replicated version is now 2.53.4. Airgap customers can download the latest version of Replicated from this [url](https://install.terraform.io/airgap/latest.tar.gz).
+
+## APPLICATION LEVEL FEATURES:
+
+1. Added the ability to [archive configuration versions](https://www.terraform.io/cloud-docs/workspaces/configurations#archiving-configuration-versions) either automatically or manually in order to free up storage space and limit the amount of sensitive data stored in Terraform Enterprise.
+2. Improved the performance and reliability when destroying workspaces.
+3. Changed documentation link on preparing a module repository.
+4. Private providers are now supported in the private registry.
+5. Added API endpoints for [downloading configuration versions](https://www.terraform.io/cloud-docs/api-docs/configuration-versions#download-configuration-files).
+1. Added [Variable Sets](https://www.terraform.io/cloud-docs/api-docs/variable-sets).
+1. Added html and/or plain text versions to all account-related emails.
+
+## APPLICATION LEVEL BUG FIXES:
+
+1. Fixed response code for JSON parse errors.
+2. Fixed API dasherizing state output keys when the top level output type is an array containing maps.
+3. Fixed 500 error when fetching admin organization that does not exist.
+1. Fixed syntax error when setting a Terraform variable to `null`.
+1. Fixed Tag creation dates that are now displayed correctly.
```

Comment:
In general, I'd want to see a bit more context on these. For example - where was JSON parsing failing? What was wrong with tag creation dates?